roqoqo/src/noise_models/overrotation.rs

```diff
@@ -0,0 +1,442 @@
+    theta_mean: f64,
+    /// Standard deviation of Gaussian distribution around mean value of theta
+    theta_std: f64,
+}
+
+#[cfg(feature = "json_schema")]
+impl schemars::JsonSchema for SingleQubitOverrotationDescription {
+    fn schema_name() -> String {
+        "SingleQubitOverrotationDescription".to_string()
+    }
+
+    fn json_schema(gen: &mut schemars::gen::SchemaGenerator) -> schemars::schema::Schema {
+        <SingleQubitOverrotationDescriptionSerialize>::json_schema(gen)
+    }
+}
+
+#[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
+#[derive(Debug, Clone, PartialEq)]
+#[cfg_attr(
+    feature = "json_schema",
+    derive(schemars::JsonSchema),
+    schemars(deny_unknown_fields)
+)]
+struct SingleQubitOverrotationDescriptionSerialize {
+    /// Rotation gate name.
+    gate: String,
+    /// Mean of Gaussian distribution to sample overrotation angle.
+    theta_mean: f64,
+    /// Standard deviation of Gaussian distribution to sample overrotation angle.
+    theta_std: f64,
+}
+
+#[cfg(feature = "serialize")]
+impl From<SingleQubitOverrotationDescription> for SingleQubitOverrotationDescriptionSerialize {
+    fn from(value: SingleQubitOverrotationDescription) -> Self {
+        let gate: String = value.gate;
+        let theta_mean: f64 = value.theta_mean;
+        let theta_std: f64 = value.theta_mean;
+
+        SingleQubitOverrotationDescriptionSerialize {
+            gate,
+            theta_mean,
+            theta_std,
+        }
+    }
+}
+
+#[cfg(feature = "serialize")]
+impl From<SingleQubitOverrotationDescriptionSerialize> for SingleQubitOverrotationDescription {
+    fn from(value: SingleQubitOverrotationDescriptionSerialize) -> Self {
+        let gate: String = value.gate;
+        let theta_mean: f64 = value.theta_mean;
+        let theta_std: f64 = value.theta_mean;
+
+        SingleQubitOverrotationDescription {
+            gate,
+            theta_mean,
+            theta_std,
+        }
+    }
+}
+
+impl SupportedVersion for SingleQubitOverrotationDescription {
+    fn minimum_supported_roqoqo_version(&self) -> (u32, u32, u32) {
+        (1, 10, 0)
+    }
+}
+
+impl SingleQubitOverrotationDescription {
+    /// Creates a new SingleQubitOverrotationDescription.
+    ///
+    /// # Arguments
+    ///
+    /// * `gate` - The name of the gate.
+    /// * `theta_mean` - The mean of Gaussian distrbution from which overrotation angle is sampled.
+    /// * `theta_std` - The standard deviation of Gaussian distrbution from which overrotation angle is sampled.
+    ///
+    /// # Returns
+    ///
+    /// `Self` - New description for overrotation noise model.
+    pub fn new(gate: &str, theta_mean: f64, theta_std: f64) -> Self {
+        SingleQubitOverrotationDescription {
+            gate: gate.to_string(),
+            theta_mean,
+            theta_std,
+        }
+    }
+
+    /// Return gate name of SingleQubitOverrotationDescription.
```

Comment:
These `Return` in this file should all be `Returns`, to be in keeping with Rust convention (which is the opposite of python/pyo3 convention, where we always want to have `Return` instead of `Returns`)